Armory Split

Geevies opened the pull request:
* The armory has been split into two levels, as it's been getting cramped lately. Much of the lethal weaponry has been moved upstairs.
* In addition to the moved weapons, the upper armory received; 4 9mm signal pistols, and 4 boot knives.
* The lower armory received; 5 .45 flash rounds, 5 .45 rubber rounds, two generic storage closets (for prototype or looted weapons), 1 box of spare handcuffs (1 was moved), 2 boxes of firing pins, and a button to open and close it from the inside.

Screenshots:
Lower Armory (button to open pictured at bottom, the two blue lockers at the top are the generic storage lockers)
![image](https://user-images.githubusercontent.com/22774890/86464092-13b59000-bd2f-11ea-93a1-71e0bfb991e2.png)

Upper Armory (note the missing hazard stripes has been fixed)
![image](https://user-images.githubusercontent.com/22774890/86464151-35167c00-bd2f-11ea-8025-ef61dddd1c40.png)

A Bad Warden (don't be this guy)
![image](https://user-images.githubusercontent.com/22774890/86464191-42336b00-bd2f-11ea-9709-50033b7ff2ed.png)

Geevies commented:
!review

BoryaTheSlayer commented:
This is wonderful. The area feels much more spacious now and I do appreciate the little new additions to the equipment.

alsoandanswer commented:
Feel it could use some tables or something. Hell, even some reinforced glass windows and windoors, with some cool floor decals too.

fernerr commented:
To me it looks quite unneccesarily big and empty, like some giant rooms with some stuff in it. Reminds me of a shop at a mall that's going out of business rather than an actual armory.
I think interesting room design is important, and this doesn't elate enough of the kind of feeling I'd expect from a key location such as this. Plus, sequestering the lethal stock on another level makes it all that much easier to ransack from the outside with little hindrance, and makes it quite a long walk for the warden to serve up those fresh guns at the front desk.
I don't like being a downer but that's my honest feedback. As always I do admire your efforts, with this and everything.

SueTheCake commented:
I agree with Ferner; the space is too huge and too unused. It feels empty and weirdly put together, and isolating the entire lethal armory on the second level will just make it too much of a weak point.

OneOneThreeEight commented:
There is at least room to fill stuff in, actually, so I don't think is such a bad thing. Good place to store future combat mechs in, maybe have some ammo lathes put in at a later date if they get coded in.

alsoandanswer commented:
> 
> 
> There is at least room to fill stuff in, actually, so I don't think this is such a bad thing. Good place to store future combat mechs in, maybe have some ammo lathes put in at a later date if they get coded in.

That's just wishful thinking at best. We can add more room _if_ they come anyway, so I don't see the point.

Arrow768 commented:
I also have to concur that it is unnecessarily large and that there are some questionable design choices.
![image](https://user-images.githubusercontent.com/1331699/86521048-f0ccce00-be4b-11ea-953d-99a813b19924.png)

- The rechargers are hidden behind the staircase (red)
- There is a lot of empty space (green)
- The armor is all over the place (orange)

The same goes for the upper level, and I do not see a reason to supply security with, what appears to be combat  or tactical knives.

Geevies commented:
alright, i adjusted the things following feedback

Upper Armory: Smaller, put racks with keypad maglocks in to make it more filled.
![image](https://user-images.githubusercontent.com/22774890/86595141-91300900-bf98-11ea-866d-ced2ba5875cd.png)

Lower Armory: Moved stairs upwards, moved the rechargers to the side, put the armour together
![image](https://user-images.githubusercontent.com/22774890/86595228-b3c22200-bf98-11ea-98a2-76d91f5bc560.png)

Geevies commented:
@Arrow768 the open space is intentional, as when the blast doors are open, there tends to be people inside, so there has to be space for them to stand. additionally, the knives are boot knives, which have a force of 6, which is 1 point of damage better than punching, if you ignore the benefits of being sharp.
